src/commands/command-docs.json

```diff
@@ -32,7 +32,7 @@
                             "type": "string"
                         },
                         "since": {
-                            "description": "the Redis version that added the command (or for module commands, the module version).",
+                            "description": "the server verion that added the command (or for module commands, the module version).",
```

Comment:
```suggestion
                            "description": "the server version that added the command (or for module commands, the module version).",
```

Reply:
Yeah I misspelled in the mass replace. :sleepy: